src/pages/unsubscribe/index.tsx

```diff
@@ -0,0 +1,190 @@
+    }
+`
+const ConfirmButton = styled(Button)`
+    width: 80px;
+    height: 40px;
+    font-size: 14px;
+    border-radius: 4px;
+    margin: 14px;
+`
+const SuccessCard = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 40px;
+    gap: 16px;
+    min-width: 330px;
+    min-height: 174px;
+    background: #ffffff;
+    border-radius: 8px;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 30px;
+
+    @media ${device.tablet} {
+        min-width: 265px;
+        min-height: 144px;
+        padding: 32px;
+        gap: 16px;
+    }
+`
+const StyledSpinner = styled.svg`
+    animation: rotate 1s linear infinite;
+    margin: 40px;
+    width: 200px;
+    height: 200px;
+
+    & .path {
+        stroke: var(--color-red-5);
+        stroke-linecap: round;
+        animation: dash 1.5s ease-in-out infinite;
+    }
+
+    @keyframes rotate {
+        100% {
+            transform: rotate(360deg);
+        }
+    }
+    @keyframes dash {
+        0% {
+            stroke-dasharray: 1, 150;
+            stroke-dashoffset: 0;
+        }
+        50% {
+            stroke-dasharray: 90, 150;
+            stroke-dashoffset: -35;
+        }
+        100% {
+            stroke-dasharray: 90, 150;
+            stroke-dashoffset: -124;
+        }
+    }
+`
+const Spinner = () => (
+    <StyledSpinner viewBox="0 0 50 50">
+        <circle className="path" cx="25" cy="25" r="20" fill="none" strokeWidth="2" />
+    </StyledSpinner>
+)
+
+const UnsubscribePage = () => {
+    const [complete_status, setCompleteStatus] = useState(false)
+    const [loading, setLoading] = useState(false)
+
+    const query = queryParams.get('hash') || ''
+    const unsubscribe_hash = decode(query).split('+')
+    const binary_user_id = unsubscribe_hash[0]
+    const checksum = unsubscribe_hash[1]
+
+    const deriv_api = useDerivApi()
+    const { send } = deriv_api
+
+    const UnsubscribeAPICall = () => {
+        setLoading(true)
+        send(
+            {
+                unsubscribe_email: 1,
+                binary_user_id: binary_user_id,
+                checksum: checksum,
+            },
+            (response) => {
+                if (!response.error) {
+                    setLoading(false)
+                    setCompleteStatus(true)
+                }
+                if (response.error) {
+                    navigate('https://app.deriv.com/account/personal-details')
+                }
+            },
+        )
+    }
+    return (
+        <Layout>
+            {loading && (
+                <UnsubscribeWrapper>
+                    <Spinner />
+                </UnsubscribeWrapper>
+            )}
+            {!loading && (
+                <UnsubscribeWrapper>
+                    {complete_status ? (
+                        <SuccessCard>
+                            <img src={CheckIcon} alt="sucess" width={48} height={48} />
+                            Unsubscribe successfully
```

Comment:
don't we need translation for this string? 
I'm not sure, you can just ignore this if it's irrelevant.

Reply:
we need translation for this, This is overlooked thanks @michio-rework

Reply:
Done 🙇